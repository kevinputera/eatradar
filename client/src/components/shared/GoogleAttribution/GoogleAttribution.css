.google-attribution {
  position: relative;
  top: 3px;
  height: 13px;
  margin: 5px 0;
}
