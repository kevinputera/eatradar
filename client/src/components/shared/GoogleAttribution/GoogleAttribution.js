import React from 'react';

import './GoogleAttribution.css';
import attributionImg from './powered_by_google_on_white_hdpi.png';

function GoogleAttribution(props) {
  return (
    <a
      href="https://www.google.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        src={attributionImg}
        alt="Powered by Google"
        className="google-attribution"
      />
    </a>
  );
}

export default GoogleAttribution;
