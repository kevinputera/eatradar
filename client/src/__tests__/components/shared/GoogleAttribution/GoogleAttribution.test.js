import React from 'react';
import { create } from 'react-test-renderer';

import GoogleAttribution from '../../../../components/shared/GoogleAttribution/GoogleAttribution';

describe('tests for GoogleAttribution component', () => {
  test('render must match snapshot', () => {
    const tree = create(<GoogleAttribution />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
