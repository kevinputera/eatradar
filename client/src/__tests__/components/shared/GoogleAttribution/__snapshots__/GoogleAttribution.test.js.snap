// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for GoogleAttribution component render must match snapshot 1`] = `
<a
  href="https://www.google.com/"
  rel="noopener noreferrer"
  target="_blank"
>
  <img
    alt="Powered by Google"
    className="google-attribution"
    src="powered_by_google_on_white_hdpi.png"
  />
</a>
`;
